include/usb/usb_device.h

```diff
@@ -183,15 +183,15 @@ struct usb_cfg_data {
 			      enum usb_dc_status_code cb_status,
 			      const uint8_t *param);
 	/** USB interface (Class) handler and storage space */
-	struct usb_interface_cfg_data interface;
+	struct usb_interface_cfg_data request_handlers;
```

Comment:
Actually, would it make sens to rename struct usb_interface_cfg_data to struct usb_interface_handlers or something more relevant that "cfg_data".

Reply:
Yes, that's right. I forgot to include that in this patch. `usb_interface_handlers` does not convince me 100%. I would like to avoid interface at all here as this could be misleading taking into consideration USB interface meaning.

Reply:
I don't have better idea than naming it `usb_request_handlers`.

Reply:
it's a better name yes